LPD-29155 API headless fixes

chestofwonders opened the pull request:
This PR contains fixes for product review -> https://liferay.atlassian.net/browse/LPD-16054

- Select `headless-admin-user/v.01` application thrown an error
- `data-set-manager/data-sets` application don't show preselected values and thrown an error
- Select `data-set-manager/data-sets` application and use `id` as key and label doesn’t show default values and trigger errors
- `data-set-manager/data-sets` application and use dates as keys and labels shows default values but trigger errors
- Selecting a preselected value crashes the modal
- Fixes Preselected Value selection and storage
- Controls /openapi.json requests to prevent unnecessary ones
- Controls getFields operations only when REST Schema has changed, to prevent errors when REST Application changes and Schema doesn't

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-source-format/9619/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/9619/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9f66856ed9d25029d096b0efcd5927358e8c0000">9f66856ed9d25029d096b0efcd5927358e8c0000</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/chestofwonders/liferay-portal/tree/LPD-25944">LPD-25944</a><br/>Branch GIT ID: <a href="https://github.com/chestofwonders/liferay-portal/commit/b11ca5ef25e73055f52f37a7b8a31a95a1083d8f">b11ca5ef25e73055f52f37a7b8a31a95a1083d8f</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-source-format/9619/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/9619/jenkins-report.html">here</a>.</h5><pre>     [exec] warning Resolution field "webpack@5.89.0" is incompatible with requested version "webpack@5.76.0"
     [exec] success Already up-to-date.
     [exec] Done in 1.14s.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat
     [exec] yarn run v1.13.0
     [exec] \$ node-scripts format --check
     [exec] Running preflight...
     [exec] 📜 Validating tsconfig files...
     [exec] ℹ️ A total of 12 CPUs were detected: launching tsc in groups of 12 projects
     [exec] ℹ️ Going to check 1 modified projects
     [exec] 🕵️ Checking apps/frontend-data-set/frontend-data-set-views-web
     [exec] Running format...
     [exec] Formatting 3 files
     [exec] Error: Format checked 3 files
     [exec] 2 files have problems
     [exec] 
     [exec]     at format (file:///opt/dev/projects/github/liferay-portal/modules/_node-scripts/format/format.mjs:276:9)
     [exec]     at async main (file:///opt/dev/projects/github/liferay-portal/modules/_node-scripts/format/index.mjs:18:2)
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] error Command failed with exit code 1.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] Gradle build finished at 2024-06-20 10:04:36.549.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:3 actionable tasks: 2 executed, 1 up-to-date
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] 
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] &gt; Run with --info or --debug option to get more log output.
     [exec] &gt; Get more help at https://help.gradle.org.
     [exec] 
     [exec] * Exception is:
     [exec] WARNING: The following functionality has been deprecated and will be removed in the next major release of the Develocity Gradle plugin. Run with '-Ddevelocity.deprecation.captureOrigin=true' to see where the deprecated functionality is being used. For assistance with migration, see https://gradle.com/help/gradle-plugin-develocity-migration.org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec] - The deprecated "gradleEnterprise.buildScan.tag" API has been replaced by "develocity.buildScan.tag"</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-source-format/9619/">test-portal-source-format#9619</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/9619/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4270">liferay-frontend#4270</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756/build/22121259">[master] ci:test:sf - chestofwonders &gt; liferay-frontend - PR#4270 - 2024-06-20[03:01:14]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>22121259<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/37019/">publish-testray-report#37019</a><br/></div></div>

chestofwonders commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-32.liferay.com/job/test-portal-source-format/8671/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/8671/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9f66856ed9d25029d096b0efcd5927358e8c0000">9f66856ed9d25029d096b0efcd5927358e8c0000</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/chestofwonders/liferay-portal/tree/LPD-25944">LPD-25944</a><br/>Branch GIT ID: <a href="https://github.com/chestofwonders/liferay-portal/commit/8c60eec2fe60e5a887bb178adab07ab9eb496939">8c60eec2fe60e5a887bb178adab07ab9eb496939</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-source-format/8671/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/8671/jenkins-report.html">here</a>.</h5></details></html>

juanjofgliferay reviewed (approved):
LGTM :rocket:

markocikos commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/SelectionFilter.tsx:
```diff
@@ -126,7 +126,7 @@ function Body({
 
 	async function getAPIValues(source: string) {
-		const response = await fetch(`/o${source}`);
+		const response = await fetch(`${source}`);
```
`fetch(source)`?

chestofwonders replied:
Done!

markocikos commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/SelectionFilter.tsx:
```diff
@@ -426,6 +438,16 @@ function Body({
 		}
 	}, [preselectedValueInput, source, sourceType]);
 
+	useEffect(() => {
+		if (selectedRESTApplication && selectedRESTEndpoint) {
+			setSource(
+				`/o${selectedRESTApplication.replace('v1.0/', '')}${selectedRESTEndpoint}`
+			);
+
+			setSourceValidationError(false);
+		}
+	}, [selectedRESTApplication, selectedRESTEndpoint]);
```
Can we avoid using `useEffect` here? We should be able to update source and validation on events that change selections. If there is a lot of code duplication, we could have a small private util.

chestofwonders replied:
Done!

markocikos commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/source_type/ApiRestApplication.tsx:
```diff
@@ -341,7 +340,9 @@ function ApiRestApplication({
 		}
-	}, [selectedRESTApplication, selectedRESTSchema]);
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [selectedRESTSchema]);
```
This is a bit scary, as it leads to stale state. Probably the only acceptable way to have this lint exception is when you want something done only once, when there are no dependencies in the list.

What is the purpose of removing this dependency? Could it be achieved in some other way?

chestofwonders replied:
The thing here is the hierarchy of data. To select a new rest application leads to have a new rest schema, but we need to calculate fields for a different schemas under the same rest application. Not removing the dependency of `selectedRESTApplication` leads to use the wrong `selectedRESTSchema` for fields fetching.

One way to remove the useEffect/dependency stuff is to invoke an util in every place `selectedRESTSchema` changes but, personally, I prefer having it as side effects decoupled from UI stuff.

markocikos replied:
> To select a new rest application leads to have a new rest schema, but we need to calculate fields for a different schemas under the same rest application.

Well, yes. The issue is, removing `selectedRESTApplication` from the list of dependencies will results in no re-calculation on selection of REST application. `getFields` within `useEffect` will not trigger. This will lead to stale `fields` state, matching the previously selected REST application.

markocikos replied:
What I would highly recommend doing here is removing `useEffect`, like you mentioned above.

markocikos reviewed (commented):
@chestofwonders few nits and questions inline

markocikos reviewed (commented):
@chestofwonders Does the selection logic really work? It seems like a bug to me :thinking:

chestofwonders commented:
@markocikos I've removed all `useEffect` from `ApiRestApplication.tsx` and moved api request for endpoint values from `SelectionFilter.tsx` to `ApiRestApplication.tsx`  👍 All inputs seems to be refreshed properly

markocikos commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/source_type/ApiRestApplication.tsx:
```diff
@@ -100,11 +107,65 @@ function ApiRestApplication({
 		return true;
 	};
 
+	async function getAPIValues(source: string) {
```
Can we put this inline in `getSourceItems`, to avoid naming?

chestofwonders replied:
sure, done!

markocikos reviewed (commented):
@chestofwonders Looks good, only some minor nits line

markocikos commented:
@chestofwonders There is a failing test in master on this feature. Please make sure that the test is green after the PR.

chestofwonders commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/SelectionFilter.tsx:
```diff
@@ -294,35 +281,20 @@ function Body({
+		setSource(source);
+		setSourceValidationError(false);
+
+		return source;
```
done!

markocikos reviewed (approved):
@chestofwonders LGTM! :sparkles:

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/10207/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10207/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-source-format/5603/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/5603/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4e06e6b171685b736a4a467fc85713906f1a086b">4e06e6b171685b736a4a467fc85713906f1a086b</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/chestofwonders/liferay-portal/tree/LPD-25944">LPD-25944</a><br/>Branch GIT ID: <a href="https://github.com/chestofwonders/liferay-portal/commit/e339d2f276465a6ea136cfc810d8c2a6d72e859e">e339d2f276465a6ea136cfc810d8c2a6d72e859e</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/5603/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/5603/jenkins-report.html">here</a>.</h5><pre>     [exec] ℹ️ A total of 12 CPUs were detected: launching tsc in groups of 12 projects
     [exec] ℹ️ Going to check 1 modified projects
     [exec] 🕵️ Checking apps/frontend-data-set/frontend-data-set-views-web
     [exec] 📝 Running format...
     [exec] ℹ️ Only running against files changed between 'master' and 'master-temp-1719821857906' branches. Use flag '--all' to run against all files.
     [exec] 📝 Formatting 5 files
     [exec] Error: Format checked 5 files
     [exec] 2 files have problems
     [exec] 
     [exec]     at format (file:///opt/dev/projects/github/liferay-portal/modules/_node-scripts/format/format.mjs:304:9)
     [exec]     at async main (file:///opt/dev/projects/github/liferay-portal/modules/_node-scripts/format/index.mjs:22:2)
     [exec] error Command failed with exit code 1.
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] Gradle build finished at 2024-07-01 08:20:40.756.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] 3 actionable tasks: 2 executed, 1 up-to-date
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] &gt; Run with --info or --debug option to get more log output.WARNING: The following functionality has been deprecated and will be removed in the next major release of the Develocity Gradle plugin. Run with '-Ddevelocity.deprecation.captureOrigin=true' to see where the deprecated functionality is being used. For assistance with migration, see https://gradle.com/help/gradle-plugin-develocity-migration.
     [exec] &gt; Get more help at https://help.gradle.org.
     [exec] 
     [exec] * Exception is:
     [exec] 
     [exec] - The deprecated "gradleEnterprise.buildScan.tag" API has been replaced by "develocity.buildScan.tag"
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec] - The deprecated "gradleEnterprise.buildScan.buildFinished" API has been replaced by "develocity.buildScan.buildFinished"  at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$1(ExecuteActionsTaskExecuter.java:148)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/5603/">test-portal-source-format#5603</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/5603/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4270">liferay-frontend#4270</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756/build/28628827">[master] ci:test:sf - chestofwonders &gt; liferay-frontend - PR#4270 - 2024-07-01[01:17:07]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>28628827<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/22137/">publish-testray-report#22137</a><br/></div></div>

markocikos commented:
@chestofwonders Did you check if there are conflicts with https://github.com/brianchandotcom/liferay-portal/tree/master? My [renaming PR](https://github.com/brianchandotcom/liferay-portal/pull/151839) is already merged there.

chestofwonders commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-33.liferay.com/job/test-portal-source-format/3287/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/3287/jenkins-report.html">here</a>.</b></p>

chestofwonders commented:
> @chestofwonders Did you check if there are conflicts with https://github.com/brianchandotcom/liferay-portal/tree/master? My [renaming PR](https://github.com/brianchandotcom/liferay-portal/pull/151839) is already merged there.

@markocikos Yes, there are conflicts 😭 so I'll wait for your PR to reach master

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4e06e6b171685b736a4a467fc85713906f1a086b">4e06e6b171685b736a4a467fc85713906f1a086b</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/chestofwonders/liferay-portal/tree/LPD-25944">LPD-25944</a><br/>Branch GIT ID: <a href="https://github.com/chestofwonders/liferay-portal/commit/f81036658a7b08428cdc55a0ba1a6e5b02857600">f81036658a7b08428cdc55a0ba1a6e5b02857600</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-33.liferay.com/job/test-portal-source-format/3287/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/3287/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-33.liferay.com/job/test-portal-source-format/3287/">test-portal-source-format#3287</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/3287/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4270">liferay-frontend#4270</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756/build/28629043">[master] ci:test:sf - chestofwonders &gt; liferay-frontend - PR#4270 - 2024-07-01[01:38:02]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>28629043<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-33.liferay.com/job/publish-testray-report/13733/">publish-testray-report#13733</a><br/></div></div>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/10207/">test-portal-acceptance-pullrequest(master)#10207</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10207/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4270">liferay-frontend#4270</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756/build/28632297">[master] ci:test:relevant - chestofwonders &gt; liferay-frontend - PR#4270 - 2024-07-01[01:17:04]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>28632297<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/21597/">publish-testray-report#21597</a><br/></div></div>

chestofwonders commented:
ci:report:28632297

liferay-continuous-integration commented:
<h1>Build completed.</h1><p>Testray CSV has been generated successfully for testrayBuildID: <b>28632297</b>.</p><p>Job Link: <a href="https://test-1-26.liferay.com/job/generate-testray-csv/194/">generate-testray-csv</a></p><p>Testray CSV Link: <a href="https://test-1-26.liferay.com/userContent/jobs/generate-testray-csv/builds/194/testray-results-28632297.csv">testray-results-28632297.csv</a></p>

markocikos commented:
@chestofwonders The date range bug in https://github.com/liferay-frontend/liferay-portal/pull/4270#issuecomment-2199655419 may be the one fixed in https://github.com/brianchandotcom/liferay-portal/pull/151839/commits/cfbb0884b4d9d0ac249ba08ac8c751d4a7b02c7d

chestofwonders commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-36.liferay.com/job/test-portal-source-format/3619/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/3619/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 10 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d66cc464ba0724b676b62463bd2b2a28a6d9b402">d66cc464ba0724b676b62463bd2b2a28a6d9b402</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/chestofwonders/liferay-portal/tree/LPD-25944">LPD-25944</a><br/>Branch GIT ID: <a href="https://github.com/chestofwonders/liferay-portal/commit/94b0fb5559c767d370425466537024307545ea2b">94b0fb5559c767d370425466537024307545ea2b</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-36.liferay.com/job/test-portal-source-format/3619/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/3619/jenkins-report.html">here</a>.</h5></details></html>

chestofwonders commented:
ci:test:relevant

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-36.liferay.com/job/test-portal-source-format/3619/">test-portal-source-format#3619</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/3619/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4270">liferay-frontend#4270</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756/build/30108963">[master] ci:test:sf - chestofwonders &gt; liferay-frontend - PR#4270 - 2024-07-03[03:06:35]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>30108963<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-36.liferay.com/job/publish-testray-report/14257/">publish-testray-report#14257</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/9998/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/9998/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 31 out of 31 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 41 out of 41 jobs passed in 1 hour 51 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d66cc464ba0724b676b62463bd2b2a28a6d9b402">d66cc464ba0724b676b62463bd2b2a28a6d9b402</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6018936ec6dbb6d685a971a4336612219224bfd4">6018936ec6dbb6d685a971a4336612219224bfd4</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/34754/routines/174485/build/30009722">EE Development Acceptance (master) - 639 - 2024-07-02[16:39:53]</a></p><details><summary><strong>ci:test:stable - 31 out of 31 jobs PASSED</strong></summary><details><summary><strong>31 Successful Jobs:</strong></summary><ul><li>build-iw-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906723/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/925742/">central-requirements-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046468/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929048/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906729/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/925743/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626580/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/994578/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/4</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929049/">js-unit-jdk8/0/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929050/">js-unit-jdk8/0/1</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046469/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906725/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929044/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046470/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626577/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906728/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046473/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626578/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046471/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906727/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929043/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626576/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906726/">modules-compile-jdk8/0/12</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046474/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626579/">modules-integration-mysql57-jdk8_stable/0/1</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046472/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>playwright-compile-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046467/">playwright-compile-jdk8/0/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929045/">playwright-js-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929047/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046466/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626572/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></details></details><details><summary><strong>ci:test:relevant - 41 out of 41 jobs PASSED</strong></summary><details><summary><strong>41 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/9998/">test-portal-acceptance-pullrequest(master)</a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906723/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/925742/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-mysql57-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046465/">empty-osgi-core-dir-mysql57-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046468/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929048/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906729/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/925743/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626580/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/994578/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/4</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929049/">js-unit-jdk8/0/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929050/">js-unit-jdk8/0/1</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046469/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906725/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929044/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046470/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626577/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906728/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046473/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626578/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046471/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906727/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929043/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626576/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906726/">modules-compile-jdk8/0/12</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046475/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/994579/">modules-integration-mysql57-jdk8/0/1</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046474/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626579/">modules-integration-mysql57-jdk8_stable/0/1</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626573/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626574/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046472/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>patching-tool-mysql57-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626575/">patching-tool-mysql57-jdk8/0/0</a></li></ul></li><li>playwright-compile-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046467/">playwright-compile-jdk8/0/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1148096/">playwright-js-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1148097/">playwright-js-tomcat90-mysql57-jdk8/1/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929045/">playwright-js-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/929047/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/906724/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1046466/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/626572/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></details></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/9998/jenkins-report.html">here</a>.</h5></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-29/jobs/test-portal-acceptance-pullrequest(master)/builds/9998/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-29/jobs/test-portal-acceptance-pullrequest(master)/builds/9998/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-29/jobs/test-portal-acceptance-pullrequest(master)/builds/9998/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/9998/">test-portal-acceptance-pullrequest(master)#9998</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/9998/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4270">liferay-frontend#4270</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/34754/routines/34756/build/30143978">[master] ci:test:relevant - chestofwonders &gt; liferay-frontend - PR#4270 - 2024-07-03[03:28:24]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>30143978<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/22174/">publish-testray-report#22174</a><br/></div></div>

chestofwonders commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
- `ci:test:relevant`
- `ci:test:sf`
- `ci:test:stable`

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-37.liferay.com/job/forward-pullrequest/1946//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/151994
[Console](https://test-1-37.liferay.com/job/forward-pullrequest/1946//consoleFull)
